presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -13,25 +13,48 @@
  */
 package com.teradata.prestomanager.agent;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 import static java.lang.Runtime.getRuntime;
 import static java.lang.String.format;
 
-// TODO: Add logger
 public final class CommandExecutor
 {
+    private static final Logger LOGGER = LogManager.getLogger(CommandExecutor.class);
+
     private CommandExecutor() {}
 
     public static int executeCommand(String command)
             throws PrestoManagerException
     {
+        LOGGER.debug("Command to be executed: {}", command);
+
         try {
             Process process = getRuntime().exec(command);
+            String output = new BufferedReader(new InputStreamReader(process.getInputStream()))
+                    .lines().collect(Collectors.joining(System.getProperty("line.separator")));
+            String error = new BufferedReader(new InputStreamReader(process.getErrorStream()))
+                    .lines().collect(Collectors.joining(System.getProperty("line.separator")));
+            LOGGER.info("Standard output from command : {} \n{}", command, output);
+            if ("".equals(error)) {
+                LOGGER.info("No standard error from command : {}", command);
+            }
+            else {
+                LOGGER.error("Standard error from command : {} \n{}", command, error);
+            }
             process.waitFor(2, TimeUnit.MINUTES);
-            return process.exitValue();
+            int retValue = process.exitValue();
+            LOGGER.debug("Process exited with return value: {}", retValue);
```

Comment:
probably want more than debug info if the process returned with a non-zero error code.  Maybe throw a presto manager exception including the error code and the stderr output.